ultralist/screen_printer.go

```diff
@@ -38,9 +38,10 @@ func (f *ScreenPrinter) Print(groupedTodos *GroupedTodos, printNotes bool) {
 		for _, todo := range groupedTodos.Groups[key] {
 			f.printTodo(todo)
 			if printNotes {
+				white := color.New(color.FgWhite)
```

Comment:
I'm wondering if it makes sense to define these colors as consts at the top, since we use them throughout this file.  What are your thoughts?

Reply:
Yes, makes sense. But therefore we need some alignment on the general use of the variables. 

We have definitions like:
`color.New(color.FgCyan).SprintFunc()`
and
`red := color.New(color.FgRed)`

Reply:
I've committed some changes in the way how I'd do it.